core/new-gui/src/app/workspace/component/result-panel/result-panel.component.ts

```diff
@@ -156,9 +187,25 @@ export class ResultPanelComponent {
       this.displayErrorMessage(`execution doesn't have any results`);
       return;
     }
+    this.executeWorkflowService.updateResultMap(response);
```

Comment:
Echo a  comment above, move `updateResultMap` inside the service. this component only calls `get`